js/indicators/apo.src.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+import H from '../parts/Globals.js';
+import '../parts/Utilities.js';
+import requiredIndicatorMixin from '../mixins/indicator-required.js';
+
+var EMA = H.seriesTypes.ema,
+    error = H.error,
+    parentLoaded = requiredIndicatorMixin.isParentIndicatorLoaded;
+
+H.seriesType('apo', 'ema',
+    /**
+     * Absolute Price Oscillator. This series requires the `linkedTo` option to
+     * be set and should be loaded after the `stock/indicators/indicators.js`
+     * and `stock/indicators/ema.js`.
+     *
+     * @extends plotOptions.ema
+     * @product highstock
+     * @sample {highstock} stock/indicators/apo
+     *                     Absolute Price Oscillator
+     * @since 7.0.0
+     * @excluding
+     *             allAreas,colorAxis,joinBy,keys,stacking,
+     *             showInNavigator,navigatorOptions,pointInterval,
+     *             pointIntervalUnit,pointPlacement,pointRange,pointStart
+     * @optionparent plotOptions.apo
+     */
+    {
+        /**
+         * Paramters used in calculation of Absolute Price Oscillator
+         * series points.
+         * @excluding period
+         */
+        params: {
+            /**
+             * Periods for Absolute Price Oscillator calculations.
+             * @default [10, 20]
+             * @type {Array}
+             * @since 7.0.0
+             */
+            periods: [10, 20]
```

Comment:
Should we use array of periods instead of two separate periods like in MACD -> https://github.com/highcharts/highcharts/blob/master/js/indicators/macd.src.js#L40 ?

In this case it will be periodSPE and periodLPE.

Add short info in docs, which periods are.

@pawelfus what do you think?

Reply:
For example Stochastic has `periods`, `macd` has multiple periods, so we don't have strict rules here..

Reply:
Ok clear!